.gitlab-ci.yml

```diff
@@ -18,6 +18,9 @@ variables:
   OPAM_VARIANT: ""
   GIT_DEPTH: "10"
 
+artifacts:
+  expire_in: 3 days
```

Comment:
Why not 1 week and remove the redundant settings?

Reply:
I prefer the information to be more "local".

Reply:
Will fix too.

Reply:
Oh actually the global setting doesn't work:
```
jobs:artifacts config contains unknown keys: expire_in
You can also test your .gitlab-ci.yml in CI Lint
```